spec/openapi.yaml

```diff
@@ -1525,6 +1525,31 @@ paths:
         
                 "We are authorized to use the Sender ID with a valid use case."
           - ```label``` name of the sender address.
+        ## Request a Sender Address from sub-accounts
```

Comment:
Just need an extra line break before this line so the header will show up correctly

Reply:
yeah, thanks, it was updated